backend/ecs_tasks/delete_files/s3.py

```diff
@@ -237,6 +233,18 @@ def delete_old_versions(client, input_bucket, input_key, new_version):
         raise DeleteOldVersionsError(errors=[str(e)])
 
 
+@retry(
+    wait=wait_exponential(multiplier=1, min=1, max=10),
+    stop=stop_after_attempt(10),
+    retry=(retry_if_result(lambda r: len(r.get("Errors", [])) > 0)),
+    retry_error_callback=lambda r: r.outcome.result(),
```

Comment:
What's your thoughts on adding a log statement in case of retry? I think it would be nice to facilitate troubleshooting especially for the situations where the retries won't fix: https://github.com/jd/tenacity#before-and-after-retry-and-logging

Reply:
I'm all for it. I'll add an after= log.